backend/files/controllers/auth/login-with-token.js

```diff
@@ -0,0 +1,25 @@
+const Account = require('../../models/Account')
```

Comment:
@h4ck3rk3y you can ignore all the files inside `backend/files` because I copy and paste all of these from [here](https://github.com/BenElferink/MERN-template)